src/azul/__init__.py

```diff
@@ -1226,6 +1226,14 @@ class GitStatus(TypedDict):
     def _git_status_env(self) -> dict[str, str]:
         return {'azul_git_' + k: str(v) for k, v in self.git_status.items()}
 
+    @property
+    def _deployment_env(self) -> dict[str, str]:
+        # This variable is distributed across many components throughout the
+        # Azul ecosystem, making it difficult to rename. It's needed for
+        # :meth:`Config.Deployment.is_shared`, which is called by a Lambda
+        # function while partitioning sources.
```

Comment:
```suggestion
        # This variable is needed for
        # :meth:`Config.Deployment.is_shared`, which is called by a Lambda
        # function while partitioning sources. Because the variable is lower case it won't be exported to the function.  Renaming it would be the obvious solution but it is distributed across many components throughout the
        # Azul ecosystem, making it difficult to rename. Instead, we export it explicitly.
```

Reply:
I tweaked the sentence order to make the diff cleaner, since the comment is added over the course of two commits.